docs/vagrant.md

```diff
@@ -23,45 +23,43 @@ Although Vagrant supports [VirtualBox](https://www.virtualbox.org/), [Hyper-V](h
 1. Verify installation by running `vagrant -v` in a terminal session.
 1. Install [VirtualBox](https://www.virtualbox.org/).
 1. If you have not already done so, fork repo to your local machine.
-1. Create a new folder to contain your `Vagrant project`. Start a terminal session in that folder.
-1. Install a [box](https://developer.hashicorp.com/vagrant/tutorials/getting-started/getting-started-boxes). The `Vagrantfile` in this repository uses `generic/ubuntu2204`. To install it run:
+1. Next you will need to install a [box](https://developer.hashicorp.com/vagrant/tutorials/getting-started/getting-started-boxes). The `Vagrantfile` in this repository uses box: `generic/ubuntu2204`. So to install this box, run in a terminal window:
 
 ```bash
-vagrant box add generic/ubuntu2204
+    vagrant box add generic/ubuntu2204
```

Comment:
You mentioned virtualbox before, but it would be good to mention it again here so people don't get confused when vagrant asks them which provider to use.

Reply:
I believe this only occurs when you use command `vagrant init`.  When I retested  on 2023-10-28 following my latest instructions, vagrant did not ask me which provider to use.